waffle/tests/test_management.py

```diff
@@ -28,6 +28,15 @@ def test_create(self):
         self.assertEqual(list(flag.groups.values_list('name', flat=True)),
                          ['waffle_group'])
 
+    def test_not_create(self):
+        """ The command shouldn't create a new flag if the create flag is
+        not set.
+        """
+        with self.assertRaisesRegexp(CommandError, "This flag doesn't exist."):
```

Comment:
Applicable to all of your new tests:

1. Use single quotes to maintain consistency.
2. Add an assertion that actually ensures the flag/switch/sample does not exist.

Reply:
Fwiw, I strongly prefer double quotes when the string has apostrophes in it, to avoid awkward escape sequences:

```py
'don\'t do this if it can be avoided'
"this's better"
'but this is best'
```